src/mono/wasm/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
```

Comment:
You don't need this file. If you don't include this file, `Directory.Build.props` in the above directory will be included by default.

Reply:
Removing .props/.targets doesn't seem to work. According to the docs, its imported by
Microsoft.Common.props, do I need to import that instead ?

Reply:
Oh, I thought this props where part of `WasmAppBuilder` project. The reason why this is not working on that project is because `wasm.proj` is not an SDK project:

https://github.com/dotnet/runtime/pull/35566/files/e7e0edc1ee2b97809844e2213574894674170d5f

You can achieve this by changing the `wasm.proj` to be an SDK project. I recommend using the Microsoft.Build.NoTargets SDK and that way you wouldn't need to have a `Directory.Build.props` nor `Directory.Build.targets` in this folder. You can do that by changing the project declaration in wasm.proj to:

`<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="Build">`

Reply:
Done, thanks.